standup/status/models.py

```diff
@@ -189,11 +193,24 @@ def htmlify(self):
         # Remove icky stuff.
         formatted = bleach.clean(self.content, tags=[])
 
+        # linkify urls
+        formatted = urlize(formatted, trim_url_limit=32)
+
         # Linkify "bug #n" and "bug n" text.
         formatted = BUG_RE.sub(
             r'<a href="http://bugzilla.mozilla.org/show_bug.cgi?id=\2">\1 \2</a>',
             formatted)
 
+        # Wrap tags in a span for formatting
+        checked = set()
+        for tag in TAG_RE.findall(formatted):
+            if tag in checked:
+                continue
```

Comment:
I don't think I understand what `checked` is doing. It seems to get set, then checked, but it doesn't seem to be modified in this for block, so won't it always be empty?